Targets/AppUIKit/Sources/Utils.swift

```diff
@@ -1,13 +1,22 @@
+let exampleSdJwt =
+    "eyJhbGciOiJFZERTQSJ9.eyJfc2RfYWxnIjoic2hhLTI1NiIsIkBjb250ZXh0IjpbImh0dHBzOi8vd3d3LnczLm9yZy9ucy9jcmVkZW50aWFscy92MiIsImh0dHBzOi8vcHVybC5pbXNnbG9iYWwub3JnL3NwZWMvb2IvdjNwMC9jb250ZXh0LTMuMC4zLmpzb24iXSwidHlwZSI6WyJWZXJpZmlhYmxlQ3JlZGVudGlhbCIsIk9wZW5CYWRnZUNyZWRlbnRpYWwiXSwiY3JlZGVudGlhbFN1YmplY3QiOnsiaWRlbnRpdHkiOlt7Ii4uLiI6ImxoeTloT3p6eWZZVFc0WWlockc4VVZRcHhFcWN1Nk9icFdvOVBhNExFRzAifSx7Imhhc2hlZCI6ZmFsc2UsImlkZW50aXR5SGFzaCI6ImpvaG4uc21pdGhAZXhhbXBsZS5jb20iLCJpZGVudGl0eVR5cGUiOiJlbWFpbEFkZHJlc3MiLCJzYWx0Ijoibm90LXVzZWQiLCJ0eXBlIjoiSWRlbnRpdHlPYmplY3QifV0sImFjaGlldmVtZW50Ijp7Im5hbWUiOiJUZWFtIE1lbWJlcnNoaXAiLCJ0eXBlIjoiQWNoaWV2ZW1lbnQifX0sImlzc3VlciI6eyJpZCI6ImRpZDpqd2s6ZXlKaGJHY2lPaUpGVXpJMU5pSXNJbU55ZGlJNklsQXRNalUySWl3aWEzUjVJam9pUlVNaUxDSjRJam9pYldKVU0yZHFPV0Z2T0dOdVMyODBNMHByY1ZSUFVtTkpRVkk0TUZnd1RVRlhRV05HWXpadlIxSk1ZeUlzSW5raU9pSmlPRlZPWTBoRE1tRkhRM0oxU1RaMFFsUldTVlkwZFc1WldFVnlTME00WkRSblJURkdaMHMwUTA1SkluMCMwIiwibmFtZSI6Ildvcmtmb3JjZSBEZXZlbG9wbWVudCBDb3VuY2lsIiwidHlwZSI6IlByb2ZpbGUifSwiYXdhcmRlZERhdGUiOiIyMDI0LTA5LTIzVDE4OjEyOjEyKzAwMDAiLCJuYW1lIjoiVGVhbU1lbWJlcnNoaXAifQ.0zGx7-Fkio8bS4EZW_odUu3F6FG7nvRVEbGKIte2OZPXGM12XEhTJptty-1ZUgPAtP_jiyU_KTP3hPrGKKNnBg~WyJoS25GV0g0R3J5dEdVMkFLUFNJSDdRIix7Imhhc2hlZCI6ZmFsc2UsImlkZW50aXR5SGFzaCI6IkpvaG4gU21pdGgiLCJpZGVudGl0eVR5cGUiOiJuYW1lIiwic2FsdCI6Im5vdC11c2VkIiwidHlwZSI6IklkZW50aXR5T2JqZWN0In1d~"
 
-let jsonstring = "{\"@context\":[\"https://www.w3.org/2018/credentials/v1\",\"https://w3id.org/security/data-integrity/v2\"],\"id\":\"https://issuer.spruceid/credentials/12345\",\"type\":[\"VerifiableCredential\"],\"issuer\":{\"id\":\"did:key:[...]\",\"image\":\"data:image/\"},\"identifier\":\"12345\",\"name\":\"Drivers License\",\"description\":\"Department of Motor Vehicles\",\"valid\":true,\"issuanceDate\":\"2024-07-10T15:37:15.566Z\",\"expirationDate\":\"2029-12-03T12:19:15.522Z\",\"credentialSubject\":{\"id\":\"did:example:123321\",\"type\":[\"DL\",\"Person\"],\"firstName\":\"Andrew\",\"lastName\":\"Carter\",\"gender\":\"Male\",\"image\":\"data:image/png;base64,i\",\"birthDate\":\"02/17/1996\"},\"proof\":[{\"type\":\"DataIntegrityProof\",\"created\":\"2024-07-10T15:37:15Z\",\"verificationMethod\":\"did:key:zDnaewRaK7eoB9KczSWkm25rVDQhun6oDhNAWkiTCpzTSTSbf#zDnaewRaK7eoB9KczSWkm25rVDQhun6oDhNAWkiTCpzTSTSbf\",\"cryptosuite\":\"ecdsa-rdfc-2019\",\"proofPurpose\":\"assertionMethod\",\"proofValue\":\"z45AXzbvdcvzbzEbbQVKBEoeLZGzLGUPHHJafATVP4HAoRTm6xdmhZbxqSyxW8FW2dCVNkdnKkV3p8sVoLzSCwBhR\"},{\"id\":\"urn:uuid:248ed8f9-e478-49ed-a551-a6cccbb6be67\",\"type\":\"DataIntegrityProof\",\"created\":\"2024-07-10T15:37:16Z\",\"verificationMethod\":\"did:key:zDnaewRaK7eoB9KczSWkm25rVDQhun6oDhNAWkiTCpzTSTSbf#zDnaewRaK7eoB9KczSWkm25rVDQhun6oDhNAWkiTCpzTSTSbf\",\"cryptosuite\":\"ecdsa-sd-2023\",\"proofPurpose\":\"assertionMethod\",\"proofValue\":\"u2V0AhVhABXMk6SgCu6l-AkHocn65wTk6o__\"}]}"
+let trustedDids = ["did:web:410b-2601-645-4580-56b0-d4-a5db-e916-a2ed.ngrok-free.app:oid4vp:client"]
```

Comment:
Please use the deployed companion service URL until we have a delegated verifier service available.

```
https://companion.ler-sandbox.spruceid.xyz/
```